bundle/edu.gemini.spdb.rollover.servlet/src/main/scala/edu/gemini/rollover/servlet/RolloverObservation.scala

```diff
@@ -48,9 +50,25 @@ object RolloverObservation {
       targetEnv  <- Option(dataObj.getTargetEnvironment)
       science    <- Option(targetEnv.getBase)
       name       <- Option(science.getTarget.getName)
-      ra         <- Option(science.getTarget.getRaDegrees)
-      dec        <- Option(science.getTarget.getDecDegrees)
-    } yield RolloverTarget(name, new Angle(ra, Angle.Unit.DEGREES), new Angle(dec, Angle.Unit.DEGREES))
+    } yield {
+
+      // Amazingly this is easier in Java
+      val when = o.getDataObject
```

Comment:
Well, the `RolloverTarget` now has an optional coordinates pair, which shows up as `<ra>0.0</ra>` in the response. We could change ITAC to handle the option in a better way but I don't see a better immediate solution.